docs/content/guides/getting-started/grid-size.md

```diff
@@ -256,7 +256,12 @@ ReactDOM.render(<ExampleComponent />, document.getElementById('example'));
 :::
 
 
-## Related API reference
+## Related articles
+
+- [Column widths](@/guides/columns/column-width.md)
+- [Row heights](@/guides/rows/row-height.md)
```

Comment:
I think that the "Row heights" guide could be extended with the advice from https://github.com/handsontable/handsontable/issues/2822#issuecomment-1304655263 

I mean to add there information how to make Handsontable aware of the correct row heights in your configuration. If you know that your rows won't fit in the default height (23 px), and you don't fix that using `rowHeights` or `autoRowSize`, you will run into problems described in #2822, #3215, #9276, #8896, #6513, etc.

Reply:
Done